jetbrains-core/src/software/aws/toolkits/jetbrains/utils/execution/steps/CliBasedStep.kt

```diff
@@ -64,6 +64,7 @@ abstract class CliBasedStep : Step() {
 
     private class CliOutputEmitter(private val messageEmitter: MessageEmitter) : ProcessAdapter() {
         override fun onTextAvailable(event: ProcessEvent, outputType: Key<*>) {
+            LOG.debug { event.text.trim() }
```

Comment:
Should we mark if it came from stdout or stderr? Also nice this was an annoying part of our tests

Reply:
uhhh, ya. Will add it